cg/meta/store/balsamic.py

```diff
@@ -57,36 +59,55 @@ def _create_analysis(bundle_data, case_obj, status, version_obj):
 
 
 def _reset_analysis_action(case_obj):
+    """Resets desired action on the case"""
     case_obj.action = None
 
 
-def _add_analysis(config_stream, case_obj):
+def parse_created(config_data: dict) -> dt.datetime:
```

Comment:
This should also be made available to the whole package. See suggestion in HK: https://github.com/Clinical-Genomics/housekeeper/blob/master/housekeeper/date.py